Update Supasquad link (#15367)
supabase/supabase: apps/www/pages/supasquad.mdx

### Builder

> Build Supabase examples, a blog, and documentation and contribute them to the [main Supabae repo](https://github.com/supabase/supabase).

Reviewer: ```suggestion
Build Supabase examples, blog about them, and add them to the [Supabase repo](https://github.com/supabase/supabase/tree/master/examples).
```
